UefiPayloadPkg/UefiPayloadPkg.fdf

```diff
@@ -208,6 +208,10 @@ INF PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcatRealTimeClockRuntimeDxe.inf
   INF MdeModulePkg/Universal/Variable/RuntimeDxe/VariableSmmRuntimeDxe.inf
 !endif
 
+!if $(SECURE_BOOT_ENABLE) == TRUE
+  INF SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigDxe.inf

```

Comment:
This always adds SecureBootConfigDxe into the main UefiPayload FV. I'm assuming that the purpose of the SECURITY_FV is for UPL, to allow builders to override features at the security/network/BDS scope. Shouldn't we follow the example of BdsDxe, for example, and not include it in the main FV in the case of UPL?

Reply:
Suggestion (by @benjamindoron ):
```
!if $(UNIVERSAL_PAYLOAD) == FALSE
  !if $(SECURE_BOOT_ENABLE) == TRUE
    INF SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigDxe.inf
  !endif
!endif
```